code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -34,7 +34,7 @@
 				tally += E.get_tally()
 	else
 		if(shoes)
-			tally += shoes.slowdown
```

Comment:
this is definitely bad, you shouldn't be factoring in a hardcoded define here. It should already be factored in when determining the variables on the shoes